src/sentry/static/sentry/app/types/index.tsx

```diff
@@ -1791,7 +1791,12 @@ export type ResolutionStatusDetails = {
   inRelease?: string;
   inNextRelease?: boolean;
   pendingEvents?: number;
+  info?: {
```

Comment:
This + pendingevents seems like it's in the wrong type entirely? Because the issue is not resolved, so ResolutionStatus does not make sense to me